extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -48,65 +25,80 @@ public static function getTemplates() {
 	 *
 	 * @param $theme
 	 * @param $layout
+	 * @param $accentColor
+	 * @param $accentColorText
 	 * @return string - infobox HTML
 	 */
-	public function renderInfobox( array $infoboxdata, $theme, $layout ) {
-		wfProfileIn( __METHOD__ );
-
-		$helper = new PortableInfoboxRenderServiceHelper();
-		$infoboxHtmlContent = '';
-		$heroData = [ ];
-
-		// decide on image width
-		$this->imagesWidth = $helper->isMobile() ?
-			PortableInfoboxRenderServiceHelper::MOBILE_THUMBNAIL_WIDTH :
-			// if europa go with bigger images! else default size
-			$helper->isEuropaTheme() ? PortableInfoboxRenderServiceHelper::EUROPA_THUMBNAIL_WIDTH :
-				PortableInfoboxRenderServiceHelper::DEFAULT_DESKTOP_THUMBNAIL_WIDTH;
-
-		foreach ( $infoboxdata as $item ) {
-			$data = $item[ 'data' ];
-			$type = $item[ 'type' ];
-
-			switch ( $type ) {
-				case 'group':
-					$infoboxHtmlContent .= $this->renderGroup( $data );
-					break;
-				case 'navigation':
-					$infoboxHtmlContent .= $this->renderItem( 'navigation', $data );
-					break;
-				default:
-					if ( $helper->isMobile() && $helper->isValidHeroDataItem( $item, $heroData ) ) {
-						$heroData[ $type ] = $data;
-						continue;
-					}
-
-					if ( $helper->isTypeSupportedInTemplates( $type, self::getTemplates() ) ) {
-						$infoboxHtmlContent .= $this->renderItem( $type, $data );
-					};
-			}
-		}
+	public function renderInfobox( array $infoboxdata, $theme, $layout, $accentColor, $accentColorText ) {
+		$helper = $this->getRenderHelper();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$helper" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)